networking/networking_operators/aws_load_balancer_operator/install-aws-load-balancer-operator.adoc

```diff
@@ -11,3 +11,8 @@ The AWS Load Balancer Operator deploys and manages the AWS Load Balancer Control
 include::modules/installing-aws-load-balancer-operator.adoc[leveloffset=+1]
 
 include::modules/installing-aws-load-balancer-operator-cli.adoc[leveloffset=+1]
+
+After installing the AWS Load Balancer Operator, you can create the AWS Load Balancer Controller.
```

Comment:
Same re: move this sentence into the module below it (or just delete the sentence).

Reply:
ty